test/views/sections/items/ItemProducerSizeViewSpec.scala

```diff
@@ -29,9 +31,12 @@ import play.api.test.FakeRequest
 import views.html.sections.items.ItemProducerSizeView
 import views.{BaseSelectors, ViewBehaviours}
 
-class ItemProducerSizeViewSpec extends SpecBase with ViewBehaviours {
+class ItemProducerSizeViewSpec extends SpecBase with ViewBehaviours with ItemFixtures {
 
-  object Selectors extends BaseSelectors
+  object Selectors extends BaseSelectors{
+    val returnToDraftLink: String = "#save-and-exit"
```

Comment:
This seems to be available in the BaseSelectors:
https://github.com/hmrc/emcs-tfe-create-movement-frontend/blob/e6802b34f95574e15ce403289ba152acd11f0a4a/test/views/BaseSelectors.scala#L37